app/pages/reporter/application/[applicationId]/version/[versionNumber]/view.tsx

```diff
@@ -72,6 +72,19 @@ class ViewApplication extends Component<Props> {
     }
   `;
 
+  constructor(props) {
+    super(props);
+    const latestSubmittedRevision =
+      props.query.application?.latestSubmittedRevision?.versionNumber;
+    const latestDraftRevision =
+      props.query.application?.latestDraftRevision?.versionNumber;
+    // Disabling in order to initialize state based on props:
+    // eslint-disable-next-line react/state-in-constructor
+    this.state = {
+      newerDraftExists: latestDraftRevision > latestSubmittedRevision
+    };
```

Comment:
we should be able to put the state initialization outside of the constructor (since the props are available before the constructor call from what I understand), this way we can remove the eslint warning

``` javascript
state = {
  newerDraftExists: latestDraftRevision > latestSubmittedRevision
}
constructor(props) { 
...
```

Reply:
> the props are available before the constructor call from what I understand

So you are right! Sometimes I get stuck in a function component mindset and forget what's available in class components as they seem so much more inflexible...

Fixing, since I'd rather not ts-ignore things.